pkg/services/notification/validation.go

```diff
@@ -10,30 +10,23 @@ import (
 	"strconv"
 
 	"openpitrix.io/logger"
+	"openpitrix.io/notification/pkg/constants"
+	"openpitrix.io/notification/pkg/gerr"
 	"openpitrix.io/notification/pkg/pb"
-	"openpitrix.io/openpitrix/pkg/gerr"
 )
 
 func ValidateSetServiceConfigParams(ctx context.Context, req *pb.ServiceConfig) error {
 	email := req.GetEmailServiceConfig().GetEmail().GetValue()
 	err := VerifyEmailFmt(ctx, email)
 	if err != nil {
-		logger.Errorf(ctx, "Failed to validateSetServiceConfigParams [%s], %+v", email, err)
+		logger.Errorf(ctx, "Failed to validate email [%s]: %+v", email, err)
 		return err
 	}
 
-	//displayEmail := req.GetEmailServiceConfig().GetDisplayEmail().GetValue()
-	//err = VerifyEmailFmt(ctx, displayEmail)
-	//if err != nil {
-	//	logger.Errorf(ctx, "Failed to validateSetServiceConfigParams [%s], %+v", displayEmail, err)
-	//	return err
-	//}
-
-	portStr := req.GetEmailServiceConfig().GetPort().GetValue()
-	portNum, err := strconv.ParseInt(portStr, 10, 64)
-	err = VerifyPortFmt(ctx, portNum)
+	port := req.GetEmailServiceConfig().GetPort().GetValue()
+	err = VerifyPortFmt(ctx, int32(port))
```

Comment:
no need to change

Reply:
port is uinit32, need to change type to init32.